src/app/data/project-data.ts

```diff
@@ -180,27 +180,29 @@ export const projectsData = [
     },
     {
         name: "ACNH.Guide",
-        description: "User-friendly display of data for bugs, fish, villagers, items, and more",
-        languages: ["en"],
+        description: "A Free Multi-Purpose Guide and Cataloging app with Catalog Scanner integration for New Horizons.",
+        languages: ["en", "es", "de", "fr", "it", "nl", "ja", "zh", "ko", "ru"],
         logo: "acnh-guide.png",
```

Comment:
Did you want to use the PNG or GIF here? If you want to use the PNG, then you probably want to leave the `logoStyle` as `icon` so the app name appears alongside it; if you want to use the GIF, please upload a much smaller version (~400px wide max) as right now the GIF is 6 MB. Whichever file is not being used should be removed.

Reply:
Im sorry, I could have sworn I removed the png file. As for the gif, I will try and make it smaller. Either way, the logo I submitted includes our name so would it still need to be icon format?

Reply:
No worries! If you use the GIF, then you can leave it as `banner` 👍